app/models/space.rb

```diff
@@ -35,6 +35,7 @@ class Space < ApplicationRecord # rubocop:disable Metrics/ClassLength
 
   validates :star_rating, numericality: { greater_than: 0, less_than: 6 }, allow_nil: true
   validates :url, url: { allow_blank: true, public_suffix: true }
+  validates :title, :address, :post_address, :post_number, :lat, :lng, presence: { presence: true }
```

Comment:
Think this can be just:
```ruby
 presence: true
``` 

instead of presence: { presence: true }

Reply:
This was rubocop, i originally wrote `validates_presence_of :title, :address, :post_address, :post_number, :lat, :lng` but i agree should just be `presence: true`